corpus/corpus/settings.py

```diff
@@ -180,3 +182,18 @@
 EMAIL_USE_SSL = True
 
 USE_TAILWIND_CDN_LINK = os.getenv("LIVECYCLE") is not None
+
+SENTRY_DSN = os.environ.get("SENTRY_DSN")
+if os.getenv("ENVIRONMENT", "PRODUCTION") == "PRODUCTION":
+    if SENTRY_DSN:
+        sentry_sdk.init(
+            dsn=SENTRY_DSN,
+            # Set traces_sample_rate to 1.0 to capture 100%
+            # of transactions for performance monitoring.
+            traces_sample_rate=1.0,
+            # Set profiles_sample_rate to 1.0 to profile 100%
+            # of sampled transactions.
+            # We recommend adjusting this value in production.
```

Comment:
Do they have any guidelines for adjusting this? What exactly does this parameter do?

Reply:
so as per [this link in the docs about django python integrations](https://docs.sentry.io/platforms/python/integrations/django/) , I believe it's all about how the transaction and each error is configured to show up when trigerred, so more like customisation